server/externs/express.js

```diff
@@ -48,6 +48,12 @@ express.Response.prototype.status = function(data) {};
 
 /**
  * @constructor
+ * @return {!express.Next}
+ */
+express.Next = function() {};
```

Comment:
Is Next a kind of callback with (express.Request, express.Response, express.Next)?